src/connectors/WSController.ts

```diff
@@ -202,4 +211,28 @@ export class WSController extends SIPConnector implements IController {
             }
         }
     }
+
+    private _startReportWatchdog() {
+        this._reportReceivedTimestamp = Date.now();
+
+        this._reportWatchdogInterval = setInterval(() => {
+            const timeout = this._reportReceivedTimestamp
+                ? Date.now() - this._reportReceivedTimestamp
+                : REPORT_WATCHDOG_TIMEOUT;
+            if (timeout >= REPORT_WATCHDOG_TIMEOUT / 3) {
+                // eslint-disable-next-line no-warning-comments
+                this.onError(`WARNING! No updates received for the last ${(timeout / 1000).toFixed(1)}s`); // TODO replace with warning
```

Comment:
To avoid the need of eslint-disable-next-line no-warning-comments you can use WIP (Work In Progress) as proposed by Jonas